doc/3/controllers/document/create-or-replace/index.md

```diff
@@ -0,0 +1,55 @@
+---
+code: true
+type: page
+title: createOrReplace
+description: Creates or Replace a new document
+---
+
+# createOrReplace
+
+Creates or replace a document in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> createOrReplace(
+      final String index,
+      final String collection,
+      final String id,
+      final ConcurrentHashMap<String, Object> document)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> createOrReplace(
+      final String index,
+      final String collection,
+      final String id,
+      final ConcurrentHashMap<String, Object> document,
+      final Boolean waitForRefresh)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                            | Index                             |
+| `collection`       | <pre>String</pre>                            | Collection                        |
+| `id`               | <pre>String</pre>                            | Document ID                       |
+| `document`         | <pre>ConcurrentHashMap<String, Object></pre> | Document content                  |
+| `waitForRefresh`   | <pre>Boolean</pre>                           | If set to `true`, Kuzzle will wait for the persistence layer to finish indexing|
```

Comment:
Inconsistent type: the same option, when present in an "options" object, is of type `boolean` (see #55 and #59)

Reply:
Imho it should be `Boolean` since `boolean` cannot be `null`, so I corrected in #55 and #59